doc/tech/approach.md

```diff
@@ -0,0 +1,99 @@
+[<< Back](./README.md)
+# CNTT Approach
+
+## Table of Contents
+* [5.1 Networking and Fabric Approach](#5.1)
+  * [5.1.1 Executive Summary](#5.1.1)
+  * [5.1.2 Strategic Objectives](#5.1.2)
+  * [5.1.3 Networking Requirements](#5.1.3)
+  * [5.1.4 Initial Steps](#5.1.4)
+
+<a name="5.1"></a>
+## 5.1 Networking and Fabric Approach
+
+> _**Editor's Note:** The purpose in the subsections is to provide initial high-level content related to networking across CNTT, so as to provide general direction and help coordinate current independent networking development activities, as well as to satisfy the networking MVP deliverables for the Baldy release. The following content addresses the overall CNTT networking solution, and it is anticipated this content will be expanded and be relocated where appropriate._
+
+The basic approach to producing the initial deliverables is to:
+A) Summarize the overall CNTT Networking and Fabric Strategy
+B) Document the initial list of Objectives and Requirements
+C) Document enough initial logistical details for contributors to create coherent content
+
+Readers be aware, in parallel with the CNTT Networking Strategy, the RI team is implementing networking for use today in labs and by RC. As RI has to deliver a working network at the same time the initial networking strategy and specifications are to be delivered, it is not expected nor mandated RI networking be conformant in the immediate future. However, the RI will need to be compatible, meaning, it delivers the network connectivity required by the RA and by RC, even if it does not implement the APIs, topology, encapsulation, etc., that will ultimately be specified herein.
+
+The following subsections, Executive Summary, Strategy Objectives, Networking Requirements and Initial Approach, respectively, are intended to provide an overview of CNTT's vision for networking and intent, objectives, requirements and their supporting rationale, as well as initial development approach.
+
+The networking within an NFVI, fabric or otherwise, is an area where there is significant variability across implementations. Leaf-Spine topology is well established, however, after topology there are countless decisions an Operator needs to make. Differences arise from many aspects, for example, is the solution layer-2 or layer-3; is the routing static or dynamic; what mechanism is used for encapsulation; what mechanism is used for isolation; does it support SR-IOV; does it support DPDK; does it employ SmartNICs; does it employ distributed control or a centralized control driving a programmable fabric; and the list continues. The multitude of permutations enable NFVI architects (Operators and Suppliers) to design (or procure) a fabric/networking solution that's optimized for their needs, whether their needs are minimal, very extensive or somewhere in between.
```

Comment:
Suggest changing `The networking within an NFVI` to `The networking within a cloud infrastructure`

Reply:
All NFVI occurrences has been replaced with cloud infrastructure.